docs/source/3x/transformers_like_api.md

```diff
@@ -0,0 +1,236 @@
+Transformers-like API
+=====
+
+1. [Introduction](#introduction)
+
+2. [Supported Algorithms](#supported-algorithms)
+
+3. [Usage For Intel CPU](#usage-for-cpu)
+
+4. [Usage For Intel GPU](#usage-for-intel-gpu)
+
+5. [Examples](#examples)
+
+## Introduction
+
+Transformers-like API provides a seamless user experience of model compressions on Transformer-based models by extending [Hugging Face transformers](https://github.com/huggingface/transformers) APIs, leveraging [Intel® Neural Compressor](https://github.com/intel/neural-compressor), and replacing Linear operator with [Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch).
```

Comment:
leveraging [Intel® Neural Compressor](https://github.com/intel/neural-compressor)  is inappropriate,  Transformer-like API is part of INC.  Suggest change to： 
Transformers-like API provides a seamless user experience of model compressions on Transformer-based models by extending [Hugging Face transformers](https://github.com/huggingface/transformers) APIs, leveraging neural compressor existing weight-only quantization capability and replacing Linear operator with [Intel® Extension for PyTorch](https://github.com/intel/intel-extension-for-pytorch).